src/components/ui/chart.tsx

```
@@ -1,353 +1,328 @@
-        __html: Object.entries(THEMES)
-          .map(
-            ([theme, prefix]) => `
+    const colorConfig = Object.entries(config).filter(([, config]) => config.theme || config.color);
+
+    if (!colorConfig.length) {
+        return null;
+    }
+
+    return (
+        <style
+            dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)